TurkishFilter/sections/antiadblock.txt

```diff
@@ -20,6 +20,9 @@ yzbwp.cc,dreamfancy.org,etkisizeleman.com,www094602.com,m.pes24.com,sportbosshd3
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37036
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
 @@||canlitribun3.com^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/37620
+!+ PLATFORM(ios, ext_android_cb, ext_safari, ext_ff, ext_opera)
+@@||oyunceviri.net^$generichide
```

Comment:
`$generichide` is not enough on my end.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/62122040-1f016f80-b2c5-11e9-9e49-386317dd79f9.png)

</details>

Please add also:
```
!+ PLATFORM(ios, ext_android_cb, ext_safari, ext_ff, ext_opera)
@@||oyunceviri.net/wp-content/uploads/*.png#
```